README.md

```diff
@@ -1,5 +1,14 @@
 ## GraphStorm
-The GraphStorm is a graph machine learning (GML) framework for enterprise use cases. It simplifies the development, training and deployment of GML models for industry-scale graphs. It provides scalable training and inference pipelines for various GML models and is optimized for extremely large graphs (measured in billons of nodes and edges). To help develop SOTA models, the framework allows very large degree of customization by providing a large collection of configurations for customizing model implementations and training pipelines as well as modularization in the training pipeline that allows users to replace every component in the pipeline.
+GraphStorm is a graph machine learning (GML) framework for enterprise use cases.
+It simplifies the development, training and deployment of GML models for industry-scale graphs.
+It provides scalable training and inference pipelines for various Graph Machine Learning (GML) models
+and is optimized for extremely large graphs (measured in billons of nodes and edges).
```

Comment:
I think these two sentences are saying the same thing?

I think you can remove line 3 and keep the more descriptive sentence in lines 4-5.

Reply:
they are different. line 3 talks about the simplicity of developing, training and inference GML models in the context of industry graphs.
line 4 and 5 emphasize the scalability.